source/includes/steps-development-mode-realm-schema.yaml

```
@@ -0,0 +1,91 @@
+title: Enable ``Development Mode``
+ref: enable-development-mode
+content: |
+
+  You can alter or create a Realm Object Model through your mobile
+  client SDK. Changes to your Realm Object Model will only be allowed
+  when :guilabel:`Development Mode` is on in the MongoDB Realm UI.
+  MongoDB Realm will reflect these changes to your Realm Object Model in
+  your Realm Schema used for Atlas. To enable :guilabel:`Development
```

```suggestion
  your Realm Schema used for Atlas.
  
  To enable :guilabel:`Development
```

Let's separate context and instructions into their own paragraphs.

---

- "Changes to your Realm Object Model will only be allowed" ..._**by whom???**_ This indicates that this uses passive voice. Use an active phrase instead.
- Why aren't changes allowed if dev mode is off?